### PR TITLE
Add the competition risk assessment

### PR DESCRIPTION
**Pull request by Scarzy**
Taken from the risk assessments PR11 build artefacts as the master
branch CircleCI build is broken

### CODE REVIEW

**Comment by Scarzy** [comment]
Deliberately not linking to this page from anywhere, relying on people knowing being passed the link, e.g. in an e-mail

**Comment by PeterJCLaw** [comment]
As a general thought -- this should probably go in the docs rather than the main website.

Separately, even if the build has come from a PR (though I think I've just fixed the CI), please can we link to the commit that the build came from (and ideally also to the CI build).

**Comment by Scarzy** [comment]
The build is: 
https://app.circleci.com/pipelines/github/srobo/risk-assessments/20/workflows/ea79ff29-3166-4eec-beea-c019e81290d9/jobs/21/artifacts

**Comment by Scarzy** [comment]
> this should probably go in the docs rather than the main website

I'm not sure I agree. The risk assessment is very event specific, and cannot be deemed to just roll on from one year to the next or even one event to the next. Therefore it is as applicable to its current location as every other item in that resources folder.

The current content of the docs seems entirely targeted at the competitors, whereas only the supervisors will be interested in the risk assessment. So it doesn't really feel like it belongs there.

**Comment by PeterJCLaw** [comment]
> > this should probably go in the docs rather than the main website
> 
> The risk assessment is very event specific, and cannot be deemed to just roll on from one year to the next

This also applies to the rules, among other things which are in the docs. I think broadly I see the website repo being more marketing biased where the docs are informational. To be honest, I'm surprised that the microgames stuff are in this repo -- I had thought we'd been putting them in the docs too.

> The current content of the docs seems entirely targeted at the competitors, whereas only the supervisors will be interested in the risk assessment.

There's an existing [Team Admin](https://studentrobotics.org/docs/team_admin/) section which is specifically for supervisors. Previously it had information about user accounts, currently it just has the Discord instructions (which are admittedly written with the expectation that anyone might end up reading it).

**Comment by Scarzy** [comment]
> > > this should probably go in the docs rather than the main website
> > 
> > 
> > The risk assessment is very event specific, and cannot be deemed to just roll on from one year to the next
> 
> This also applies to the rules, among other things which are in the docs. I think broadly I see the website repo being more marketing biased where the docs are informational. 

Then basically everything in any of the events pages doesn't belong on the website. Parking maps, COVID guidelines, media consent, marathon updates etc. are informational. 

> > The current content of the docs seems entirely targeted at the competitors, whereas only the supervisors will be interested in the risk assessment.
> 
> There's an existing [Team Admin](https://studentrobotics.org/docs/team_admin/) section which is specifically for supervisors. Previously it had information about user accounts, currently it just has the Discord instructions (which are admittedly written with the expectation that anyone might end up reading it).

Ok, but equally I wouldn't necessarily want an obvious link to it. It's for volunteers, supervisors, and potentially venue and external providers, so extends beyond just team admin. If we had to create a link to it I can see far more of an argument for linking to it from the event page than from the team admin page.

File: risk-assessments.md [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+title: Risk Assessments
+date: 2022-04-16
+---
+
+Last updated: {{ page.date | date: "%e %B %Y" }}.
+
+## SR2022
+
+### Competition
+The risk assessment for the SR2022 Competition can be found [here]({{ 'resources/sr2022/risk-assessments/competition.pdf' | prepend: site.baseurl }}).
```
Comment (PeterJCLaw) [inline risk-assessments.md:12]: nit: using "here" as link text is discouraged for accessibility reasons.

This link also doesn't work due to a missing `/` in the url.

```suggestion

We have published a [risk assessment for the SR2022 Competition]({{ '/resources/sr2022/risk-assessments/competition.pdf' | prepend: site.baseurl }}).
```

File: risk-assessments.md [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+
+If you have any questions please contact us at [{{ site.emails.teams }}][teams-email].
+
+[teams-email]: mailto:{{ site.emails.teams }}
+
```
Comment (PeterJCLaw) [inline risk-assessments.md:17]: nit: trailing newline
```suggestion
```

**Review by PeterJCLaw — APPROVED** [review]

**Pull request merged**